pkg/operation/garden/garden.go

```diff
@@ -251,5 +263,48 @@ func BootstrapCluster(k8sGardenClient kubernetes.Interface, gardenNamespace stri
 	if !gardenVersionOK {
 		return fmt.Errorf("the Kubernetes version of the Garden cluster must be at least %s", minGardenVersion)
 	}
+
+	if secrets[common.GardenRoleGlobalMonitoring] == nil {
+		if secret, err := generateMonitoringSecret(k8sGardenClient, gardenNamespace); err == nil {
+			secrets[common.GardenRoleGlobalMonitoring] = secret
+		}
+		return err
+	}
+
 	return nil
 }
+
+func generateMonitoringSecret(k8sGardenClient kubernetes.Interface, gardenNamespace string) (*corev1.Secret, error) {
+
+	basicAuthSecret := &secretutils.BasicAuthSecretConfig{
+		Name:   "seed-monitoring-ingress-credentials",
+		Format: secretutils.BasicAuthFormatNormal,
+
+		Username:       "admin",
+		PasswordLength: 32,
+	}
+	basicAuth, err := basicAuthSecret.Generate()
+	if err != nil {
+		return nil, err
+	}
+
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      basicAuthSecret.Name,
+			Namespace: gardenNamespace,
+		},
+		Type: corev1.SecretTypeOpaque,
```

Comment:
Remove type and data - they should only be set in the func inside CreateOrUpdate below